Character/Character/Debug/Character.log

```diff
@@ -0,0 +1,2 @@
+﻿  character.cpp

```

Comment:
git rm -r --cached .
git add .
git commit -m "Drop files from .gitignore"

https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor/13541721